src/panel/settings/settings_desktop.vala

```diff
@@ -193,33 +189,20 @@ namespace Budgie {
 			} else if (use_desktop_type == DesktopType.DESKTOPFOLDER) { // Desktop Folder
 				return desktop_folder_settings.get_boolean("show-desktopfolder");
 			} else if (use_desktop_type == DesktopType.NEMO) { // Nemo
-				return nemo_settings.get_string("desktop-layout").has_prefix("true::");
+				return nemo_settings.get_boolean("show-desktop-icons");
```

Comment:
The reason I didn't use "show-desktop-icons" is Nemo reports it as deprecated: https://github.com/linuxmint/nemo/blob/master/libnemo-private/org.nemo.gschema.xml#L526

Reply:
They handle both values via https://github.com/linuxmint/nemo/blob/master/src/nemo-desktop-manager.c#L689 but I intentionally went with the more forward-supported one since I wouldn't want this sort of integration to potentially be put into an immediate position where the key would be deprecated and the integration break.